querydsl-sql-codegen/src/main/java/com/mysema/query/sql/codegen/MetaDataExporter.java

```diff
@@ -200,7 +203,18 @@ public void export(DatabaseMetaData md) throws SQLException {
         }
 
         String[] typesArray = null;
-        if (!exportAll) {
+
+        if (tableTypesToExport != null && !tableTypesToExport.isEmpty()) {
+            List<String> types = new ArrayList<String>();
+            if (tableTypesToExport.contains(",")) {
```

Comment:
This check seems unnecessary, since split(",") will also work if no comma is contained.